### ISSUE TITLE
Final Peer Review

### ISSUE THREAD

**Issue by sjaroslawski** [rainy0niu/Predicting-Crimes-in-Philadelphia#12]
I thought that this project had many good aspects but a few areas that were questionable.  The team did a good job of covering most of what we learned in class.  They attempted to use many different regularizers and loss functions to have enough models to test on their validation set.  They even used unsupervised learning in one of their models.  The team also did a good job explaining how well each model performed and the reasons why they thought some models performed better than others.  I thought deseasonalizing the data was a very good idea that was not necessarily covered in class.

One improvement the group could have made was taking on a slightly harder challenge.  The group predicting levels of different types of crime by month.  However, I do not believe this is all that useful to the police.  It would be much more useful for the police what times of the day and week certain crimes are likely to happen.  Also, where crimes occur would be very useful so more cops could be close to likely areas of crime.  Further, the group addresses that three types of crime are outliers, but do not seem to put forth a good solution to predicting those types of crime.  Overall, the team did a good job writing the report and utilizing many concepts from class.
